.travis.yml

```diff
@@ -2,27 +2,45 @@ language: php
 sudo: false
 
 php:
-  - 5.6
   - 7.0
   - 7.1
 
 addons:
   rethinkdb: "2.3"
 
+services:
+  - mongodb
+
+env:
+  matrix:
+    - DRIVER=rethinkdb
+    - DRIVER=mongodb
+
 cache:
   directories:
     - /home/travis/build/RoySegall/nuntius-bot/vendor
 
 before_script:
+
+  - echo $DRIVER
+  # Install MongoDB for PHP.
+  # source: https://gist.github.com/gyndav/2351174.
+  - wget http://pecl.php.net/get/mongodb-1.4.3.tgz
+  - tar -xzf mongodb-1.4.3.tgz
+  - sh -c "cd mongodb-1.4.3 && pecl install mongodb"
+  - echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
```

Comment:
Todo: 
1. Check if this is mongo environment
2. Cache the `.so` for preventing rebuild anytime.